app/routes/image-set/cv-results.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function() {
+    var imageSet = this.modelFor('imageSet');
+    return this.store.find('cvResult', {image_set_id: imageSet.get('id')});
+  },
+
+  setupController: function(controller, model) {
+    var imageSet = this.modelFor('imageSet');
+    controller.setProperties({
+      model: model,
+      searchImageSet: imageSet
```

Comment:
Yeah, that might be a bit better. It also feels fine to just call it `imageSet`, I think, if I understand the intent of this page correctly. You're viewing the cv request results for the image set specified int he URL, right? `imageSet` seems sufficient